.github/workflows/check-minimum-revision.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Check Minimum Revision"
+
+on:
+  workflow_call:
+    inputs:
+      GIT_SHA:
+        required: true
+        type: string
+
+env:
+  GIT_SHA: ${{ inputs.GIT_SHA }}
+  MINIMUM_REVISION: ${{ secrets.MINIMUM_REVISION }}
+
+jobs:
+  check-minimum-revision:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ env.GIT_SHA }}
+          fetch-depth: 1000
+      - name: Check merge base is new enough
+        id: check-merge-base
+        run: |
+          set -ex
```

Comment:
can you put this in a standalone script so that it's easier to test locally?

Reply:
Because this is a pull request target, that would then immediately fail all workflows because they wouldnt have the file